packages/select/src/Option.tsx

```diff
@@ -23,6 +24,20 @@ const optionStyle = css`
   outline: none;
   overflow-wrap: anywhere;
   transition: background-color 150ms ease-in-out;
+  position: relative;
+
+  &:before {
```

Comment:
We need to transition this element in like we do in the side navigation. IIRC it's a scale transform transition on the vertical axis combined with the opacity transition.

Reply:
Done!